trinity/main.py

```diff
@@ -43,6 +45,8 @@
     choices=LOG_LEVEL_CHOICES,
     default=DEFAULT_LOG_LEVEL,
 )
+parser.add_argument('console', nargs='?', default='')
```

Comment:
Can you update this to leverage the `add_subparser` API

https://docs.python.org/3.5/library/argparse.html#argparse.ArgumentParser.add_subparsers